src/JSInterop/Microsoft.JSInterop/src/JSInProcessObjectReference.cs

```diff
@@ -24,8 +24,7 @@ internal JSInProcessObjectReference(JSInProcessRuntime jsRuntime, long id) : bas
         /// <param name="identifier">An identifier for the function to invoke. For example, the value <c>"someScope.someFunction"</c> will invoke the function <c>someScope.someFunction</c> on the target instance.</param>
         /// <param name="args">JSON-serializable arguments.</param>
         /// <returns>An instance of <typeparamref name="TValue"/> obtained by JSON-deserializing the return value.</returns>
-        [return: MaybeNull]
-        public TValue Invoke<TValue>(string identifier, params object[] args)
+        public TValue? Invoke<TValue>(string identifier, params object[] args)
```

Comment:
Can args be null?

Reply:
I have a separate PR that changes these: https://github.com/dotnet/aspnetcore/pull/25277. I didn't want to conflate the two if I could avoid it

Reply:
It does look like this one should be:

```suggestion
        public TValue? Invoke<TValue>(string identifier, params object?[]? args)
```